ios/insporationUITests/insporationUITests.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  insporationUITests.swift
+//  insporationUITests
+//
+//  Created by Thorsten Claus on 03.11.20.
+//
+
+import XCTest
+// Mark
+class insporationUITests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+
+        // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testTakeSnapshots() throws {
+        // Assumtions:
+        // App is prepaired with an logged-in account, first tab (Main) is selected and Streams is selected
+  
+        let app = XCUIApplication()
+        app.launch()
+        setupSnapshot(app)
+        sleep(3) // Wait until stream is loaded
```

Comment:
Seems brittle, is there no way to wait until some specific element appears? We maybe could even just attach a semantic label to the loading spinner and wait for it to disappear?